subtitles.go

```diff
@@ -158,6 +160,54 @@ func (c *Color) TTMLString() string {
 	return fmt.Sprintf("%.6x", uint32(c.Red)<<16|uint32(c.Green)<<8|uint32(c.Blue))
 }
 
+// WebVTTColorString expresses the color as a webvtt string
+func (c *Color) WebVTTColorString() string {
+	switch c {
+	case ColorBlack:
+		return "<c.black>" //rgba(0,0,0,1)
+	case ColorRed:
+		return "<c.red>" //rgba(255,0,0,1)
+	case ColorGreen:
+		return "<c.lime>" //rgba(0,255,0,1)
+	case ColorYellow:
+		return "<c.yellow>" //rgba(255,255,0,1)
+	case ColorBlue:
+		return "<c.blue>" //rgba(0,0,255,1)
+	case ColorMagenta:
+		return "<c.magenta>" //rgba(255,0,255,1)
+	case ColorCyan:
+		return "<c.cyan>" //rgba(0,255,255,1)
+	case ColorWhite:
+		return "<c.white>" //rgba(255,255,255,1)
+	default:
+		return ""
+	}
+
+}
+
+// WebVTTPositionFromSTL Webvtt Justification From STL
+func (sa *StyleAttributes) WebVTTPositionFromSTL() string {
```

Comment:
See below, make this method belong to `*stlVerticalPosition`type, and rename the function to `WebVTTPosition`